src/osd/osd_types.h

```diff
@@ -3986,7 +3986,7 @@ struct object_info_t {
 
   flag_t flags;
 
-  static string get_flag_string(flag_t flags) {
+  string get_flag_string(flag_t flags) const {
```

Comment:
please note, i was referencing the `static string get_flag_string(unsigned flags);` you are changing not `string get_flag_string() const;`. and the precondition i mentioned above does not apply to the latter.